docs/network/edges.html

```diff
@@ -101,9 +101,30 @@ <h3>Options</h3>
 var options = {
   edges:{
     arrows: {
-      to:     {enabled: false, scaleFactor:1, type:'arrow'},
-      middle: {enabled: false, scaleFactor:1, type:'arrow'},
-      from:   {enabled: false, scaleFactor:1, type:'arrow'}
+      to: {
+        enabled: false,
+        imageHeight: undefined,
+        imageWidth: undefined,
+        scaleFactor: 1,
+        src: undefined,
+        type: "arrow"
+      },
+      middle: {
+        enabled: false,
+        imageHeight: 32,
+        imageWidth: 32,
+        scaleFactor: 1,
+        src: "http://placehold.it/32x32/008888",
```

Comment:
we should not depend on external links for the docs.

Reply:
There's an image from our site now.